static/app/components/workflowEngine/form.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+export const StickyFooter = styled('div')`
+  position: sticky;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  padding: ${space(2)} ${space(4)};
+  background: ${p => p.theme.background};
+  border-top: 1px solid ${p => p.theme.translucentGray200};
+  box-shadow: 0px -4px 24px 0px rgba(43, 34, 51, 0.12);
```

Comment:
any chance we have theme variables for this spacing / color / shadow?

Reply:
We actually don't! This is the default heavy shadow with the Y distance flipped. It is used in another spot, so probably worth adding to the `theme`.

Reply:
Extracted this in f2d424db9a8